code/modules/admin/holder.dm

```diff
@@ -300,6 +300,7 @@ GLOBAL_PROTECT(admin_verbs_default)
 	/datum/admins/proc/toggle_prayers,
 	/datum/admins/proc/mcdb,
 	/datum/admins/proc/check_fingerprints,
+	/datum/admins/proc/set_xeno_stat_buffs,
```

Comment:
ditto

Reply:
This one is really setting the buff

Reply:
shouldnt it be `apply_xeno_stat_buffs` then?

Reply:
no, you're inputting the buff value, then applying it. Set is more appropriate IMO

Reply:
pending put this with avdebug